### PR TITLE
Sync s3

### PR DESCRIPTION
**Pull request by peusebiu**
**What type of PR is this?**

**Which issue does this PR fix**:

**What does this PR do / Why do we need it**:

**If an issue # is not available please add repro steps and logs showing the issue**:

**Testing done on this change**:

**Automation added to e2e**:

**Will this break upgrades or downgrades?**

**Does this PR introduce any user-facing change?**:

```release-note

```

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by rchincha** [comment]
@peusebiu pls add an example as well so folks know what to try

**Comment by rchincha** [comment]
cc: @elee1766

File: pkg/extensions/config/sync/config.go [hunk 1]
```diff
@@ -15,6 +15,7 @@ type Credentials struct {
 type Config struct {
 	Enable          *bool
 	CredentialsFile string
+	TmpDir          string
```
Comment (rchincha) [inline pkg/extensions/config/sync/config.go:18]: Call this SyncDir and add a comment that this is needed only for cloud case.
Also, can we add a check in LoadConfig that this is indeed the case.

**Review by andaaron — APPROVED** [review]

**Review by andaaron — APPROVED** [review]

**Review by rchincha — APPROVED** [review]
lgtm

**Pull request merged**